### DISCUSSION TITLE
blog/swift-value-reference

### DISCUSSION THREAD

**Discussion by giscus** [theodorusclarence/theodorusclarence.com#297] [General]
# blog/swift-value-reference

In-depth explanation of Swift's value and reference types.

https://theodorusclarence.com/blog/swift-value-reference

**Comment by theodorusclarence** [comment]
Hello, kindly leave a like (or 5 😄), and comment if you got something to ask! 🚀

**Reply by zhex900** [reply]
How did you draw your diagrams? What tool did you?

**Reply by theodorusclarence** [reply]
I'm using [excalidraw](https://excalidraw.com/) 😃
